custom_theme/main.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
-
-{% block site_meta %}
+  
+  {% block site_meta %}
+    <script src="https://cmp.osano.com/AzZMxHTbQDOQD8c1J/a2e89f0e-f467-4542-bfea-30ea2c1a6648/osano.js"></script>
```

Comment:
```suggestion
{% block site_meta %}
    <script src="https://cmp.osano.com/AzZMxHTbQDOQD8c1J/a2e89f0e-f467-4542-bfea-30ea2c1a6648/osano.js" defer></script>
```

Reply:
Removing "defer"

`Important: The Osano.js should be placed without any async or defer attributes. Compliance cannot be guaranteed when using the above attributes. `

**Link:** https://docs.osano.com/installing-osano